File: BACKEND/FCUnirea.Business/Models/GamesModel.cs
//GamesModel
using System;

namespace FCUnirea.Business.Models
{
    public class GamesModel
    {
        public DateTime GameDate { get; set; }
        public int HomeTeamScore { get; set; }
        public int AwayTeamScore { get; set; }
        public int TicketsSold { get; set; }
        public string RefereeName { get; set; }
        public bool IsPlayed { get; set; }

        //--------------Relations--------------------//
        // Foreign Key
        public int? Game_CompetitionsId { get; set; }
        public int? Game_StadiumsId { get; set; }
        public int? Game_AwayTeamId { get; set; }
        public int? Game_HomeTeamId { get; set; }
    }
}

File: FRONTEND/FCUnirea/src/app/core/models/games-model.ts
export interface Game {
    id: number;
    gameDate: string;
    homeTeamScore: number;
    awayTeamScore: number;
    game_HomeTeamId: number;
    game_AwayTeamId: number;
    game_CompetitionsId: number;
    isPlayed: boolean;

    homeTeamName: string;
    awayTeamName: string;
    competitionName: string; // 👈 Nou

  }
  